tools/megahit_contig2fastg/megahit_contig2fastg.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version='1.0' encoding='utf-8'?>
+<tool id="megahit_contig2fastg" name="MEGAHIT Contig2fastg" version="@VERSION@.3">
+    <description>for converting MEGAHIT's k*.contigs.fa to fastg</description>
+    <macros>
+        <token name="@VERSION@">1.1.2</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">megahit</requirement>
+    </requirements>
+    <version_command>megahit --version</version_command>
+    <command detect_errors="exit_code"><![CDATA[
+        megahit_toolkit contig2fastg
+        '${kmer}'
+        '${contigs}' > '${fastg}'
+    ]]></command>
+    <inputs>
+        <param name="contigs" type="data" format="fastq,fastqsanger,fasta,fastq.gz,fastqsanger.gz" multiple="true" label="Single-end file(s)" help="FASTQ/FASTA/FASTQ.GZ files accepted" />
+        <param name="kmer" type="integer" value="21" label="K-mer length" />
+    </inputs>
+    <outputs>
+        <data format="fasta" name="fastg" label="Conversion to FASTG with ${tool.name} on ${on_string}" />
```

Comment:
this should be fastq, isn't it? @jraysajulga are you up to add the fastg datatype to Galaxy.

Reply:
Eventually, it should be fastg... @jj-umn is working on adding the fastg datatype to Galaxy.

Reply:
Can you wait for this, or do you want it in now ... if you need it please change it to txt for now.

Reply:
On second thought, it might take a while for fastg to be integrated. We would like to test it out soon. I changed the format from **fasta** to **txt**